docs/python-sdk/changelog.md

```diff
@@ -7,6 +7,52 @@ sidebar_position: 8
 
 # Changelog
 
+## v1.18.0 (2025-04-28)
+
+**[`fused-py`](/python-sdk/)**
+
+- `fused.submit` API has been improved.
```

Comment:
I think it would be good to explicitly list some of the relevant changes. The main one that comes to mind is the (breaking) change that it will now raise an error by default, if there is any run erroring